lib/api/backbeat/listLifecycleOrphanDeleteMarkers.js

```diff
@@ -19,15 +19,16 @@ function handleResult(listParams, requestMaxKeys, authInfo,
 
 /**
  * listLifecycleOrphanDeleteMarkers - Return list of expired object delete marker in bucket
- * @param  {AuthInfo} authInfo - Instance of AuthInfo class with
- *                               requester's info
- * @param  {object} request    - http request object
- * @param  {function} log      - Werelogs request logger
- * @param  {function} callback - callback to respond to http request
- *                               with either error code or xml response body
+ * @param {AuthInfo} authInfo            - Instance of AuthInfo class with
+ *                                         requester's info
+ * @param {array} locationConstraints    - array of location contraint
+ * @param {object} request               - http request object
+ * @param {function} log                 - Werelogs request logger
+ * @param {function} callback            - callback to respond to http request
+ *                                          with either error code or xml response body
  * @return {undefined}
  */
-function listLifecycleOrphanDeleteMarkers(authInfo, request, log, callback) {
+function listLifecycleOrphanDeleteMarkers(authInfo, locationConstraints, request, log, callback) {
```

Comment:
I believe we need to pass `locationConstraints` in the `listParams` here as well.

Reply:
Good catch. I am wondering how the functional "listing orphan delete markers" tests did not catch this. We might want to give it a quick look. Thanks.

Reply:
I don't think so. What we pass in the `listParams`  is the `excludedDataStoreName` property that will be used to exclude keys that have already been transitioned. 
It is not necessary since orphan delete markers are not transitioned.

Reply:
OK, makes sense, it would be useful to describe in the JSDoc of each listing backend function which params are supported and what they are for, to avoid surprises later if we add new params etc., otherwise one may assume that all backend functions support the same set of params, which is not the case here.